src/VisualStudio/Core/Def/SymbolSearch/VisualStudioSymbolSearchService.cs

```diff
@@ -45,8 +45,6 @@ internal partial class VisualStudioSymbolSearchService : AbstractDelayStartedSer
         private readonly string _localSettingsDirectory;
         private readonly LogService _logService;
 
-        private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
```

Comment:
very telling that we have a CTS that we never cancel or dispose.  THis was done originally to allow the impl to carry the idea of being cancellable based on some concept.  However, we didn't have anything to actually attach to.  Now we do.